pkg/updater/gcs.go

```diff
@@ -390,19 +390,44 @@ func convertResult(log logrus.FieldLogger, nameCfg nameConfig, id string, header
 
 	emailAddressesInterface, ok := result.finished.Finished.Metadata[EmailListKey]
 	if ok {
-		emailAddresses, ok := emailAddressesInterface.([]string)
-		if ok {
-			out.Column.EmailAddresses = emailAddresses
-		} else {
-			log.Error("email addresses type is not list of strings")
+		emailAddresses, err := convertToListOfStrings(emailAddressesInterface)
```

Comment:
I think you can simplify this by just declaring the []string variable and using json.Marshal to coerce the value into that. e.g.

var []string emailAddresses
if err := json.Unmarshal(emailAddressesInterface, &emailAddresses); err != nil {
  log.WithError(err).Error("<error message>")
}

Reply:
Updated